corehq/warehouse/dbaccessors.py

```diff
@@ -40,3 +42,24 @@ def _get_ids_by_last_modified(cls, doc_types, start_datetime, end_datetime):
         )
         for result in results:
             yield result['id']
+
+
+def get_synclog_ids_by_date(start_datetime, end_datetime):
+    from casexml.apps.phone.models import SyncLog
+
+    results = SyncLog.view(
+        "sync_logs_by_date/view",
+        startkey=[json_format_datetime(start_datetime)],
+        endkey=[json_format_datetime(end_datetime)],
+        reduce=False,
+        include_docs=False
+    )
+    for result in results:
+        yield result['id']
```

Comment:
interesting. right that was definitely something i didn't think about for this one. is it actually that bad if we reprocess a  sync log? shouldn't it just correctly update instead of insert?